### PR TITLE
 [zos-console] Added method to get a command response

### LINKED ISSUE #78: [zos-console] Add method to get a command response
Add a method to the Python zos-console SDK to call the z/OSMF REST API `GET /zosmf/restconsoles/consoles/<console-name>/solmsgs/C<key-number>`

* z/OSMF REST API documentation: https://www.ibm.com/docs/en/zos/2.5.0?topic=services-get-command-response
* Python zos-console SDK in the "next" branch: https://github.com/zowe/zowe-client-python-sdk/tree/next/src/zos_console
* Node.js zos-console SDK for comparison: https://github.com/zowe/zowe-cli/blob/master/packages/zosconsole/src/CollectCommand.ts

### PR DESCRIPTION
**Pull request by aadityasinha-dotcom**
Added a method to call the z/OSMF REST API `GET /zosmf/restconsoles/consoles/<console-name>/solmsgs/C<key-number>`
Signed-off-by: Aaditya Sinha [aadityasinha009@gmail.com](mailto:aadityasinha009@gmail.com)
Resolves #78

### CODE REVIEW

**Review by zFernand0 — CHANGES_REQUESTED** [review]
hey @aadityasinha-dotcom 
I see a few things that can be improved.

Here is a summary of the changes I'm thinking of:
https://github.com/aadityasinha-dotcom/zowe-client-python-sdk/compare/get_command...zowe:zowe-client-python-sdk:fix-console-apis

1. Help text change.
    - the get_response is not exactly getting any response of a console command, it's more collecting operator-type messages that are still outstanding (i.e. left to be replied/responded to)
2. `solmsgs` should be optional, and likely be renamed to `console` 
3. it seems that the `defcn` in the url is preventing us to use the given console name.

FYI, there has to be a better way to handle the `defcn` vs console-name issue 😋 
The compare URL is there to serve as a working example 😋

File: src/zos_console/zowe/zos_console_for_zowe_sdk/console.py [hunk 1]
```diff
@@ -41,24 +41,30 @@ def issue_command(self, command, console=None):
         custom_args = self._create_custom_request_arguments()
         request_body = {"cmd": command}
         custom_args["json"] = request_body
+        custom_args = self.__create_custom_request_arguments()
+        request_body = '{"cmd": "%s"}' % (command)
+        custom_args["data"] = request_body
```
Comment (zFernand0) [inline src/zos_console/zowe/zos_console_for_zowe_sdk/console.py:46]: I believe this part here might be redundant 😋

**Review by zFernand0 — COMMENTED** [review]
Thanks for addressing the requested changes.
I left one more comment.

Also, do you think we could have some tests added ?

**Comment by zFernand0** [comment]
Please address the DCO sign-off issues in addition to the changes requested.

**Review by zFernand0 — CHANGES_REQUESTED** [review]
LGTM!
Just waiting on DCO fixes and some tests! 🥳

**Comment by aadityasinha-dotcom** [comment]
Sorry will do dco been sick for few days

**Comment by aadityasinha-dotcom** [comment]
Recreated this PR https://github.com/zowe/zowe-client-python-sdk/pull/142

**Pull request closed without merging**